.github/workflows/pull-request.yml

```diff
@@ -8,10 +8,30 @@ on:
     branches:
       - main
 
+env:
+  RETENTION_DAYS: 7
```

Comment:
Is this used by the cache?

Reply:
This is used to set how long to store testing reports: playwright report, allure report, accessibility report

Reply:
Unlike Artifacts cache is limited to 7 days by default but artifacts we should limit with retention-days (at least as I know)